### PR TITLE
fix: General Fixes and Improvements from v1.0.19.2 Fixes Branch

### PR DESCRIPTION
**Pull request by Cristhianzl**
This pull request includes significant updates to the GitHub Actions workflows and Docker build configurations. The changes aim to support new build types, improve version handling, and streamline the Docker build process.

### Workflow Enhancements:
* [`.github/workflows/docker-build.yml`](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82R63): Added support for building Docker images with a new version format and additional checks for version inputs. Introduced `version-no-v` output to handle versions without a leading 'v'. [[1]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82R63) [[2]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L102-R179) [[3]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L138-L142) [[4]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L152-L163) [[5]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L172-R236) [[6]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L231-R305) [[7]](diffhunk://#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82L297-R353)
* [`.github/workflows/release.yml`](diffhunk://#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R27-R31): Added a new job to build Docker images for Langflow with an entrypoint (`main-ep`). [[1]](diffhunk://#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R27-R31) [[2]](diffhunk://#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R211-R220)
* [`.github/workflows/release_nightly.yml`](diffhunk://#diff-3e9627631213861db56751e801ce149ca7974e5559af73fb36f9216013875006R17-R21): Introduced a new job to build nightly Docker images for Langflow with an entrypoint. [[1]](diffhunk://#diff-3e9627631213861db56751e801ce149ca7974e5559af73fb36f9216013875006R17-R21) [[2]](diffhunk://#diff-3e9627631213861db56751e801ce149ca7974e5559af73fb36f9216013875006R42-R46) [[3]](diffhunk://#diff-3e9627631213861db56751e801ce149ca7974e5559af73fb36f9216013875006L200-L201) [[4]](diffhunk://#diff-3e9627631213861db56751e801ce149ca7974e5559af73fb36f9216013875006L213-R236)

### Dockerfile Updates:
* [`docker/build_and_push.Dockerfile`](diffhunk://#diff-b3bac7333e6b802f4fad29c96d5e0bda6a8b528a8919e57dd1952d3acd5f24bfR56): Added `README.md` to the Docker build context to ensure it is included in the build.
* [`docker/build_and_push_ep.Dockerfile`](diffhunk://#diff-774026a983ab9016559579119e44c81fa3554ed8293dcef6cfe60bca936891d8R1-R91): Created a new Dockerfile for building Langflow with an entrypoint, including steps for building dependencies and setting up the runtime environment.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ packages = ["src/backend/langflow"]
 
 [project]
 name = "langflow"
-version = "1.0.19"
+version = "1.0.19.post1"
```
Comment (jordanrfrazier) [inline pyproject.toml:15]: should this be post2?

Comment (jordanrfrazier) [reply pyproject.toml:15]: https://pypi.org/project/langflow/

File: pyproject.toml [hunk 2]
```diff
@@ -31,7 +31,7 @@ maintainers = [
 
 # Define your main dependencies here
 dependencies = [
-    "langflow-base==0.0.97",
+    "langflow-base==0.0.98",
```
Comment (jordanrfrazier) [inline pyproject.toml:34]: https://pypi.org/project/langflow-base/

**Comment by edwinjosechittilappilly** [comment]
LGTM

**Review by edwinjosechittilappilly — APPROVED** [review]
LGTM

**Review by italojohnny — APPROVED** [review]

**Review by jordanrfrazier — APPROVED** [review]

**Pull request merged**